.github/workflows/windows-unittests.yml

```diff
@@ -1,7 +1,11 @@
 name: "Windows unit tests"
 
 on:
-  - pull_request
+  push:
+    branches:
+      - main
+      - 7.[0-9][0-9].x
```

Comment:
This would be "broken" when we release version 8. Should be 7 replaced on [0-9]

Reply:
I just used the same triggers as can be found in https://github.com/DataDog/datadog-agent/blob/main/.github/workflows/codeql-analysis.yml

Reply:
Fair enough, I see your point (after I have expand the view)